### PR TITLE
[Fix] Fixed base showcase robot

### PR DESCRIPTION
**Pull request by PuroSlavKing**
## About the PR

The search query "ShowcaseRobot" in F5, gives out 5 robot layouts, and one of them is BaseShowcaseRobot, it is impossible to spawn it, and it has no texture, so I decided to remove it from F5.

**Media**

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: PuroSlavKing
- fix: Fixed base showcase robot in F5 (just remove it)

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Decoration/showcase.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 - type: entity
   id: BaseShowcaseRobot
+  noSpawn: true
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Decoration/showcase.yml:3]: Make it abstract rather than nospawn

Comment (PuroSlavKing) [reply Resources/Prototypes/Entities/Structures/Decoration/showcase.yml:3]: > Make it abstract rather than nospawn

"abstract: true"?

Comment (vulppine) [reply Resources/Prototypes/Entities/Structures/Decoration/showcase.yml:3]: correct

Comment (PuroSlavKing) [reply Resources/Prototypes/Entities/Structures/Decoration/showcase.yml]: > correct

done

**Review by ZoldorfTheWizard — APPROVED** [review]

**Pull request merged**